src/rendering/vcPolygonModel.cpp

```diff
@@ -84,6 +84,11 @@ struct AsyncPolygonModelLoadMeshInfo
   vcPolygonModel *pPolygonModel;
   vcPolygonModelMesh *pMesh;
   vcP3N3UV2Vertex *pVerts;
+  const vcVertexLayoutTypes* pMeshLayout;
```

Comment:
The `*` goes on the right, this seems to be done throughout this PR, you might need to update your VS settings for this.

Reply:
Sure, changes are in